src/modm/driver/motion/adns9800.lb

```diff
@@ -27,15 +28,28 @@ def prepare(module, options):
             enumeration=["a4", "a4b", "a5", "a6"],
             description="Select the firmware version to initialize the device with.",
             default="a6"))
+    module.add_option(
+        BooleanOption(
+            name="use_fiber",
+            description="Use an advanced driver without backwards compatibilty for protothreads.",
+            default="false"))
```

Comment:
Borrowed this opt-in option from `processing:protothreads` to allow taking the pure fibers driver thats also brings more features.

Reply:
nah, just depend on the `:processing:fiber` module directly and ignore backward compatibility. GCC13 refuses to compile the RF_YIELD macro, cos it's undefined behavior, so we cannot support resumables anymore. The proper way to do this would be to use C++23 coroutines, but unfortunately the APIs are different so that it cannot be done without significant code changes. Let's just use fibers, it will solve \*a lot\* of problems in the future.